java/pom.xml

```diff
@@ -1,41 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Here the file is removed....

Reply:
you are right, cant understand why after merge that file is missing